app/views/parks/index.html.erb

```diff
@@ -11,4 +11,5 @@
     </div>
   </li></a><br>
   <% end %>
+
```

Comment:
Please get rid of this added line -- be sure to carefully review the changes you're committing so that you don't commit trivial changes :)